<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintDialog</class>
 <widget class="QDialog" name="PrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayouPrint">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxPrintDeparture">
       <property name="title">
        <string>Departure Airport</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDepartureOverview">
          <property name="text">
           <string>Overview and METAR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDepartureRunways">
          <property name="text">
           <string>Runways</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDepartureSoftRunways">
          <property name="text">
           <string>Include soft Surface Runways</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDepartureDetailRunways">
          <property name="text">
           <string>Detailed Runway Information</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDepartureCom">
          <property name="text">
           <string>COM Frequencies</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDepartureAppr">
          <property name="text">
           <string>Approaches</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBoxPrintDestination">
       <property name="title">
        <string>Destination Airport</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDestinationOverview">
          <property name="text">
           <string>Overview and METAR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDestinationRunways">
          <property name="text">
           <string>Runways</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDestinationSoftRunways">
          <property name="text">
           <string>Include soft Surface Runways</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDestinationDetailRunways">
          <property name="text">
           <string>Detailed Runway Information</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDestinationCom">
          <property name="text">
           <string>COM Frequencies</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrintDestinationAppr">
          <property name="text">
           <string>Approaches</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxPrintFlightplan">
     <property name="text">
      <string>Flight Plan</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBoxPrint">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxPrintDepartureOverview</tabstop>
  <tabstop>checkBoxPrintDestinationOverview</tabstop>
  <tabstop>checkBoxPrintDepartureRunways</tabstop>
  <tabstop>checkBoxPrintDestinationRunways</tabstop>
  <tabstop>checkBoxPrintDepartureSoftRunways</tabstop>
  <tabstop>checkBoxPrintDestinationSoftRunways</tabstop>
  <tabstop>checkBoxPrintDepartureDetailRunways</tabstop>
  <tabstop>checkBoxPrintDestinationDetailRunways</tabstop>
  <tabstop>checkBoxPrintDepartureCom</tabstop>
  <tabstop>checkBoxPrintDestinationCom</tabstop>
  <tabstop>checkBoxPrintDepartureAppr</tabstop>
  <tabstop>checkBoxPrintDestinationAppr</tabstop>
  <tabstop>checkBoxPrintFlightplan</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxPrint</sender>
   <signal>accepted()</signal>
   <receiver>PrintDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxPrint</sender>
   <signal>rejected()</signal>
   <receiver>PrintDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
